Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+ADD kafka-offset-exporter /
+EXPOSE 9000
+ENTRYPOINT ["/kafka-offset-exporter"]
+CMD ["-brokers", "127.0.0.1:9092", "-topics", "^[^_].*", "-groups", "."]
```

Comment:
Should change `-groups` argument to `'.*'` otherwise no consumer-groups will match the pattern and therefore their stats will not be published but topic oldest/youngest offsets will still work, so you may not immediately notice.